lib/auth/auth.go

```diff
@@ -4358,6 +4358,23 @@ func (a *Server) SubmitUsageEvent(ctx context.Context, req *proto.SubmitUsageEve
 	return nil
 }
 
+// Ping gets basic info about the auth server.
```

Comment:
Mention here that the information returned by ping isn't supposed to be sensitive? Or copy the comment in ServerWithRoles here too?

It's easy to see this evolve to return something it shouldn't without having knowledge from the interplay between Server.Ping and ServerWithRoles.Ping. I'd guess that's why the method wasn't declared here.